--- base/logging.cc.orig 2017-01-10 06:02:59.747810600 +0300
+++ base/logging.cc      2017-01-10 05:57:12.802860300 +0300
@@ -531,7 +531,7 @@
 }

 LogMessage::~LogMessage() {
-#if !defined(OFFICIAL_BUILD) && !defined(OS_NACL) && !defined(__UCLIBC__)
+#if !defined(OFFICIAL_BUILD) && !defined(OS_NACL) && defined(__GLIBC__)
   if (severity_ == LOG_FATAL && !base::debug::BeingDebugged()) {
     // Include a stack trace on a fatal, unless a debugger is attached.
     base::debug::StackTrace trace;
