--- base/message_loop/message_pump_libevent.cc.orig 2017-01-10 06:02:59.781410400 +0300
+++ base/message_loop/message_pump_libevent.cc      2017-01-10 05:58:54.262218000 +0300
@@ -14,7 +14,7 @@
 #include "base/files/file_util.h"
 #include "base/logging.h"
 #include "base/posix/eintr_wrapper.h"
-#include "base/third_party/libevent/event.h"
+#include "event.h"
 #include "base/time/time.h"
 #include "base/trace_event/trace_event.h"
 #include "build/build_config.h"
