--- base/threading/platform_thread_linux.cc.orig 2017-01-10 06:02:59.970897300 +0300
+++ base/threading/platform_thread_linux.cc      2017-01-10 06:01:21.793423100 +0300
@@ -175,7 +175,8 @@

 size_t GetDefaultThreadStackSize(const pthread_attr_t& attributes) {
 #if !defined(THREAD_SANITIZER)
-  return 0;
+  // use 2mb to avoid running out of space
+  return (2 * 1024 * 1024);
 #else
   // ThreadSanitizer bloats the stack heavily. Evidence has been that the
   // default stack size isn't enough for some browser tests.
